MatrixSDKTests/TestPlans/UnitTestsWithSanitizers.xctestplan

```diff
@@ -63,6 +63,8 @@
         "MXQRCodeDataUnitTests",
         "MXReplyEventParserUnitTests",
         "MXResponseUnitTests",
+        "MXSyncResponseUnitTests\/testMergingArrays()",
```

Comment:
Use "MXSyncResponseUnitTests" to run all of them.

Reply:
👍